components/suggest/src/suggest.udl

```diff
@@ -26,13 +27,15 @@ interface Suggestion {
     Amp(
         string title,
         string url,
+        string raw_url,
```

Comment:
note: This is a source breaking change for Swift consumers (which we don't have, yet! 🤫), but not Kotlin or JS.